### DISCUSSION TITLE
Running workers in dev

### DISCUSSION THREAD

**Discussion by andrei0x309** [vercel/turborepo#1202] [Help] [answered]
I was wondering...  
If you have some workers in an app, that loads an absolute path .js in a worker what's the best way of having that running when you're running on dev under ts-node. 

Right now I do this: 
1 First I check if the worker files exist ( which will exists on a docker instance build with docker out) 
2 If it doesn't exist I just load those services as async functions instead of dedicated workers.

I'm pretty happy with this solution but I've been asked if there's a better one...

**Comment by anthonyshew** [comment, marked as answer]
This solution sounds fine to me, personally! In any case, Turborepo shouldn't care about how your workers are set up.
